tools/systems.r

```diff
@@ -166,15 +166,18 @@ systems: [
     0.4.61 linux-ia64/linux "libc-ia64"
         #SGD #LEN #LLC #PIP2 #LP64 <HID> /HID /DYN %M %DL
 
-    BeOS: 5
+    Haiku: 5
     ;-------------------------------------------------------------------------
     0.5.1 _ _
         ; labeled "BeOS R5 PPC" in Rebol 1.2, but "BeOS R4 PPC" in Rebol 2.5
 
     0.5.2 _ _
         ; was: "BeOS R5 iX86"
 
-    0.5.75 haiku/posix "x86-32"
+    0.5.40 haiku-x64/haiku "x64"
+        #SGD #LEN #LLC #LLP64 <DCE> /DCE %NWK
+
+    0.5.75 haiku/haiku "x86-32"
```

Comment:
The 0.5.75 version numbering was a joke based on the syllable pattern of 5-7-5 in Haiku poems.

But this hasn't really escaped into the wild anywhere.  So if there's now a 32-bit and 64-bit version, then 0.5.4 and 0.5.40 would fit the pattern better.

If this build section is renamed to Haiku, then it's probably time that any references in Ren-C sources to BeOS in #defines can be dropped at this point.